packages/react-native-reanimated/src/culori/index.ts

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+/*
+ * The vast majority of the code exported by this module is a direct copy of the code from
+ * the culori package (see https://culorijs.org/), which deserves full credit.
+ */
```

Comment:
`culori` library MIT license [asks to include](https://github.com/Evercoder/culori?tab=MIT-1-ov-file) copyright notice and permission notice in all copies or substantial portions of the Software.

Reply:
@patrycjakalinska I'm not expert, but essentially that's the same as reanimated's license itself which makes it applicable for the entire project, doesn't it? Would you feel more comfortable having their permission + copyright notice written in every file, nonetheless?

Reply:
@d4vidi i dug deeper, and I think it'll be enough to link a source and an author. So probably the comment in index file is okay, but you can add links to source files of other copied culori related functions - it'd also be easier for maintaining 😅

Reply:
Cool, will do my best 🙏🏻

Reply:
Alright, I've extended the comment to the other files under `culori/` in my latest commit - [af753a8](https://github.com/software-mansion/react-native-reanimated/pull/6782/commits/af753a8e6bc607e0bf2c385c7da58d41df49130c)